Pdf/Helper/Data.php

```diff
@@ -0,0 +1,58 @@
+<?php
+namespace MagentoJapan\Pdf\Helper;
+
+use Magento\Store\Model\ScopeInterface;
+use Magento\Framework\App\Helper\Context;
+
+/**
+ * Class Data
+ * @package MagentoJapan\Pdf\Helper
+ */
+class Data extends \Magento\Framework\App\Helper\AbstractHelper
```

Comment:
Having `Helper` classes is not required and sometimes may even cause confusion for 3rd party developers to figure out the purpose of the class. I would suggest moving configuration reading logic to a separate model, with clear name reflecting it's purpose.